imports/ui/QuestionSidebar.jsx

```diff
@@ -51,7 +53,11 @@ export class QuestionSidebar extends ControlledForm {
       })
       this.forceUpdate()
     })
-    
+
+    Meteor.call('courses.courseRequiresApprovedQuestions',this.props.courseId, (e, approved) => {
```

Comment:
Again, be explicit about course requiring only *public* questions to be approved